packages/patternfly-4/react-core/src/components/StyledSystem/StyledBox.d.ts

```diff
@@ -0,0 +1,62 @@
+import { SFC, HTMLProps, ReactNode } from 'react';
+
+export interface StyledBoxProps {
```

Comment:
These should be available from styled-system `@types/styled-system` I am still not sure the best way to handle that though. That can either be listed as a peerDependency, and optional, or a straight dependency. @priley86  might have a better idea for that.

Reply:
Okay cool, I can extend my interface with those typings. (just fixing a small issue with them here as well https://github.com/DefinitelyTyped/DefinitelyTyped/pull/30170).
I am adding it as an optional dependency for now, sound good?